rust/bindings_list

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: GPL-2.0
+
+--opaque-type xregs_state
+--opaque-type desc_struct
+--opaque-type arch_lbr_state
+--opaque-type local_apic
+
+# For some arch, arch_spinlock_t is defined as ZST which triggers Rust warning.
```

Comment:
Perhaps mention which arch triggered the warning, it may be useful later on or for someone to take a look at `bindgen`.

Reply:
I was actually referring to #263. But after some digging it turns out to be not related to a specific arch, but rather when CONFIG_SMP is disabled.